lib/yoti/sandbox/sandbox_client.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Sandbox
+  # Client is responsible for setting up test data in the sandbox instance
+  class Client
+    attr_accessor :app_id
+    attr_accessor :key
+    attr_accessor :base_url
+    attr_accessor :version_id
+
+    def initialize(app_id:, private_key:, base_url:, version_id: "v1")
+      @app_id = app_id
+      @base_url = base_url
+      @key = OpenSSL::PKey::RSA.new(Base64.decode64(private_key))
+      @version_id = version_id
+    end
+
+    def setup_sharing_profile(profile)
+      endpoint = "/apps/#{app_id}/tokens?\
+nonce=#{SecureRandom.uuid}&timestamp=#{Time.now.to_i}"
+      uri = URI(
+        "#{@base_url}/#{@version_id}#{endpoint}"
```

Comment:
Perhaps we can remove `#{@version_id}` and include `/v1` on `#{@base_url}` to simplify things?

Reply:
The v1 isn't part of the bath when creating the Sandbox session, but is when we use it.